packages/react/src/format.test.tsx

```diff
@@ -138,13 +139,23 @@ describe("formatElements", function () {
     const elements = formatElements("<div><0/><0/></div>", {
       div: <div />,
       0: <span>hi</span>,
-    }) as Array<React.ReactElement>
+    })
+
+    expect(React.isValidElement(elements)).toBe(true)
+
+    assert.ok(
```

Comment:
let's please use an `expect` statement here like we do everywhere. Thank you

Reply:
Already done as such above, but this has a underlying `asserts` satisfies the TS compiler. The alternative is casting, which isn't runtime-validated. I'm fine with either approach, but wanted to explain my choice."

Reply:
hmm, understood, thank you!

Let's see what other people have to say.

While reading the code, the assert simply stuck me as unexpected and I didn't see its purpose (We don't use it anywhere else.).
With regards to type safety, you still had to instruct `React.isValidElement` with 

```
<{
        children: React.ReactElement[]
      }>
```

which, afaict, is a form of an `as` cast because without it, TS doesn't know that `children` is an array. I think `as` is equivalent in terms of type safety tbh, and I'd rather use that because with `as` people know that it's a frowned-upon keyword.

Reply:
No problem, I removed them in favor of plain `as` casting.